Src/Support/GoogleApis/Apis/[Media]/Download/MediaDownloader.cs

```diff
@@ -166,96 +175,86 @@ public IDownloadProgress Download(string url, Stream stream)
                 throw new ArgumentException("stream doesn't support write operations");
             }
 
-            RequestBuilder builder = null;
-
-            var uri = new Uri(url);
-            if (string.IsNullOrEmpty(uri.Query))
+            // Add alt=media to the query parameters.
+            var uri = new UriBuilder(url);
+            if (uri.Query == null || uri.Query.Length <= 1)

```

Comment:
I'm deferring to [the docs for UriBuilder.Query](https://msdn.microsoft.com/en-us/library/system.uribuilder.query(v=vs.110).aspx), which say:
> Do not append a string directly to this property. If the length of Query is greater than 1, retrieve the property value as a string, remove the leading question mark, append the new query string, and set the property with the combined string.

I don't believe I actually saw just "?" in my testing.